test/dispatchers/keyDispatcherTests.ts

```diff
@@ -2,50 +2,59 @@
 
 describe("Dispatchers", () => {
   describe("Key Dispatcher", () => {
-    it("triggers callback on keydown", () => {
-      let ked = Plottable.Dispatchers.Key.getDispatcher();
-      let keyCodeToSend = 65;
-      let keyDowned = false;
-      let body = d3.select("body");
-      let callback = (code: number, e: KeyboardEvent) => {
-        keyDowned = true;
-        assert.strictEqual(code, keyCodeToSend, "correct keycode was passed");
-        assert.isNotNull(e, "key event was passed to the callback");
-      };
-
-      ked.onKeyDown(callback);
-
-      TestMethods.triggerFakeKeyboardEvent("keydown", body, keyCodeToSend);
-      assert.isTrue(keyDowned, "callback when a key was pressed");
-
-      ked.offKeyDown(callback);
-
-      keyDowned = false;
-      TestMethods.triggerFakeKeyboardEvent("keydown", body, keyCodeToSend);
-      assert.isFalse(keyDowned, "nothing happens when a key was pressed");
-    });
 
-    it("triggers callback on keyup", () => {
-      let ked = Plottable.Dispatchers.Key.getDispatcher();
-      let keyCodeToSend = 65;
-      let keyUped = false;
-      let body = d3.select("body");
-      let callback = (code: number, e: KeyboardEvent) => {
-        keyUped = true;
-        assert.strictEqual(code, keyCodeToSend, "correct keycode was passed");
-        assert.isNotNull(e, "key event was passed to the callback");
-      };
+    describe("Basic usage", () => {
+      let keyCodeToSend: number;
+      let keyDispatcher: Plottable.Dispatchers.Key;
+
+      let body: d3.Selection<void>;
+
+      beforeEach(() => {
+        keyCodeToSend = 65;
+        keyDispatcher = Plottable.Dispatchers.Key.getDispatcher();
+        body = d3.select("body");
+      });
+
+      it("calls the onKeydown callback", () => {
```

Comment:
Maybe change the test name to something like
it("can set a callback to be called while a key is pressed down")?

Mainly change the name to be more semantic and more aligned to what a user is trying to do.